docs/data/material/getting-started/templates/checkout/getCheckoutTheme.tsx

```diff
@@ -0,0 +1,816 @@
+                },
+              },
+              '& :before': {
+                borderBottom: '1px solid',
+                borderColor: gray[200],
+              },
+              '&:hover': {
+                '& :before': {
+                  borderColor: brand[300],
+                },
+              },
+            }),
+            ...(theme.palette.mode === 'dark' && {
+              '& .MuiInputBase-input': {
+                '&::placeholder': {
+                  opacity: 1,
+                  color: gray[500],
+                },
+              },
+              '& .MuiOutlinedInput-root': {
+                boxSizing: 'border-box',
+                minWidth: 280,
+                minHeight: 40,
+                height: '100%',
+                borderRadius: '10px',
+                transition: 'border-color 120ms ease-in',
+                '& fieldset': {
+                  border: 'none',
+                  boxShadow: ' 0px 2px 4px rgba(0, 0, 0, 0.4)',
+                  background: `${alpha(gray[800], 0.4)}`,
+                },
+                '&:hover': {
+                  borderColor: brand[300],
+                },
+                '&.Mui-focused': {
+                  borderColor: brand[400],
+                  outline: '4px solid',
+                  outlineColor: alpha(brand[500], 0.5),
+                },
+              },
+              ...(ownerState.variant === 'standard' && {
+                '&.MuiTextField-root': {
+                  '& .MuiInput-root:hover:not(.Mui-disabled, .Mui-error):before': {
+                    borderColor: brand[200],
+                  },
+                },
+                '& :before': {
+                  borderBottom: '1px solid',
+                  borderColor: gray[700],
+                },
+                '&:hover': {
+                  '& :before': {
+                    borderColor: brand[300],
+                  },
+                },
+              }),
+            }),
+          }),
+        },
+      },
+      MuiStepConnector: {
+        styleOverrides: {
+          line: ({ theme }) => ({
+            borderTop: '1px solid',
+            borderColor: theme.palette.divider,
+            flex: 1,
+            borderRadius: '99px',
+          }),
+        },
+      },
+      MuiStepLabel: {
+        styleOverrides: {
+          label: ({ theme }) => ({
+            '&.Mui-completed': {
+              opacity: 0.4,
+              ...(theme.palette.mode === 'dark' && { opacity: 0.3 }),
+            },
+          }),
+        },
+      },
+      MuiStepIcon: {
+        variants: [
+          {
+            props: { completed: true },
+            style: () => ({
+              width: 16,
+              height: 16,
+            }),
+          },
+        ],
+        styleOverrides: {
+          root: ({ theme }) => ({
+            color: 'transparent',
+            border: `1px solid ${gray[400]}`,
+            width: 8,
+            height: 8,
```

Comment:
I'd either make the circle small or the checked icon smaller so there's no layout shift as you progress through the steps